opensquirrel/exporter/quantify_scheduler_exporter.py

```diff
@@ -29,19 +32,32 @@ def __init__(self, qubit_register_name: str):
         self.schedule = quantify_scheduler.Schedule("Exported OpenSquirrel circuit")
 
     def visit_measure(self, g: Measure) -> None:
-        self.schedule.add(quantify_scheduler_gates.Measure(self._get_qubit_string(g.qubit)))
+        qubit_index = g.qubit.index
+        self.schedule.add(
+            quantify_scheduler_gates.Measure(
+                self._get_qubit_string(g.qubit),
+                acq_channel=qubit_index,
+                acq_index=qubit_index,
+                acq_protocol="ThresholdedAcquisition",
+            )
+        )
+        return
```

Comment:
```suggestion
        self.schedule.add(
            quantify_scheduler_gates.Measure(
                self._get_qubit_string(g.qubit),
                acq_channel=g.qubit.index,
                acq_index=g.qubit.index,
                acq_protocol="ThresholdedAcquisition",
            )
        )
        return
```

I think `g.qubit.index` is quite self explanatory to not needing to define a `qubit_index`.